rosbag2_interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.8)
+project(rosbag2_interfaces)
```

Comment:
I'd actually prefer `rosbag2_srvs` as a project name.

Reply:
I was thinking about leaving us open to potentially defining messages, or actions. Would you prefer stylistically that (if they come to be needed) those would be separate packages?